### DISCUSSION TITLE
hide some pages from fuse search

### DISCUSSION THREAD

**Discussion by grahampcharles** [razonyang/hugo-theme-bootstrap#813] [Q&A]
Hi! Is there a page Param that will remove that page from the fuse search results? Like `index = false` or similar?

Many thanks,

Graham

**Comment by razonyang** [comment]
Hi, there is no such parameter so far, I'll create a PR for this later.

**Comment by razonyang** [comment]
Implemented by #815, you'll need to [upgrade](https://deploy-preview-815--hbs-docs.netlify.app/en/docs/getting-started/upgradation/) the theme to `master` for using this feature.
